### PR TITLE
Removed AppBar from Feedback and Grade Screen

### PR DESCRIPTION
**Pull request by Joshuadholly**

### CODE REVIEW

File: lib/Views/ReportCard/ReportCard.dart [hunk 1]
```diff
@@ -143,19 +143,22 @@ class ReportCard extends StatelessWidget {
...
+                bottom:0,
+                right:0,
+                left: 300,
+                top:0,
+              child: Text(
+                  'Power of 1 Score',
+                  style: TextStyle(
+                  fontSize: 45,
+                    color: Colors.white,
+              ),
```
Comment (antpowell) [inline lib/Views/ReportCard/ReportCard.dart:159]: May cause an issue because it will always start at 300px no matter the device screen resolution

**Review by antpowell — APPROVED** [review]

File: lib/Views/ReportCard/ReportCard.dart (hunk 1 above)
Comment (antpowell) [inline lib/Views/ReportCard/ReportCard.dart:159]: These lines here

Comment (Joshuadholly) [reply lib/Views/ReportCard/ReportCard.dart:159]: Gotcha

File: lib/Views/ReportCard/ReportCard.dart [hunk 2]
```diff
@@ -143,19 +143,22 @@ class ReportCard extends StatelessWidget {
...
-          ),
-          centerTitle: true,
-        ),
         body: SafeArea(
           child: Stack(
             children: [
+              Positioned(
+                bottom:0,
+                right:0,
+                left: 300,
```
Comment (antpowell) [inline lib/Views/ReportCard/ReportCard.dart:152]: This is the specific line itself

Comment (antpowell) [inline lib/Views/ReportCard/ReportCard.dart:152]: Sorry this is the line

**Pull request merged**